File: Data Structures & Algorithms/concatenation-of-array/submission-2.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] x = new int[nums.length * 2];

        for (int i = 0; i < nums.length; i++){
            x[i] = nums[i];
            x[i + nums.length] = nums[i];
        }
        
        return x;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        
        
        count = {}

        for n in nums:
            count[n] = count.get(n, 0) + 1 

        h= [[] for i in range(len(nums) + 1)]
      
        for key, value in count.items():
            h[value].append(key)

        result = []
        for i in range(len(h) - 1, 0, -1):
            for num in h[i]:
                result.append(num)
                if len(result) == k:
                    return result
        
        
            

        